diff --git a/build/modules/access_tokens_guard/main.js b/build/modules/access_tokens_guard/main.js
index 6305607086b6d17aca1e998dabf9e52a710d9f3b..906538943f3f36d2ca0aa1c3246d64b927e43508 100644
--- a/build/modules/access_tokens_guard/main.js
+++ b/build/modules/access_tokens_guard/main.js
@@ -1196,15 +1196,15 @@ var DbAccessTokensProvider = class DbAccessTokensProvider {
 		/**
 		* Update last time the token is used
 		*/
-		dbRow.last_used_at = /* @__PURE__ */ new Date();
+		const last_used_at = /* @__PURE__ */ new Date();
 		await db.from(this.table).where({
 			id: dbRow.id,
 			type: dbRow.type
-		}).update({ last_used_at: dbRow.last_used_at });
+		}).update({ last_used_at });
 		/**
 		* Convert to access token instance
 		*/
-		const accessToken = this.dbRowToAccessToken(dbRow);
+		const accessToken = this.dbRowToAccessToken({ ...dbRow, last_used_at });
 		/**
 		* Ensure the token secret matches the token hash
 		*/
